tv_apps/smart-home/js/home.js

```diff
@@ -305,7 +305,8 @@
     onCardUpdated: function(card, idx) {
       var cardButton = this.cardScrollable.getItemFromNode(
                                               this.cardScrollable.getNode(idx));
-      CardUtil.updateCardName(cardButton, card);
+      // nextElementSibling of cardButton is the name span.
+      CardUtil.updateCardName(cardButton.nextElementSibling, card);
```

Comment:
Why not retrieve the name span by something like `this.cardScrollable.getNode(idx).querySelector('.name')`? It would be more readable than `nextElementSibling`.